src/common/ipc/ipc.ts

```diff
@@ -30,7 +30,15 @@ import { ClusterFrameInfo, clusterFrameMap } from "../cluster-frames";
 import type { Disposer } from "../utils";
 import type remote from "@electron/remote";
 
-const electronRemote = ipcMain ? null : require("@electron/remote");
+const electronRemote = (() => {
+  if (ipcRenderer) {
+    try {
```

Comment:
Hmm. Add comment to explain the underlying very good reason for this wild-card try/catch.

Reply:
This code will be going away as soon as #4625 is merged so I don't think it is really necessary.